codesign/lib/src/utils.dart

```diff
@@ -124,3 +124,11 @@ String? getValueFromEnvOrArgs(
   throw CodesignException('Expected either the CLI arg --$name or the environment variable $envName '
       'to be provided!');
 }
+
+String joinEntitlementPaths(String entitlementParentPath, String pathToJoin) {
+  if (entitlementParentPath == "") {
+    return pathToJoin;
```

Comment:
The magic empty string is confusing, I'd rather the caller be explicit about the path than abstracting it into this method.